app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt

```diff
@@ -387,8 +382,36 @@ class MainActivity : BaseActivity() {
 
     //region In-App Review
     private fun initAppReviewManager() {
+        lifecycleScope.launch {
+            inAppReviewManager.shouldDisplayReviewDialog
+                .flowWithLifecycle(lifecycle, Lifecycle.State.STARTED)
+                .collect { shouldDisplayReviewDialog ->
+                    if (shouldDisplayReviewDialog) {
+                        trackInAppReview(MatomoName.PresentAlert)
+                        inAppReviewManager.onUserWantsToDismiss()
```

Comment:
Probably a leftover, or should be "inlined" to call what the `onUserWantsToDismiss` callback calls.